lib/spack/docs/build_systems/rpackage.rst

```diff
@@ -124,6 +126,67 @@ If you only specify the URL for the latest release, your package will
 no longer be able to fetch that version as soon as a new release comes
 out. To get around this, add the archive directory as a ``list_url``.
 
+^^^^^^^^^^^^^^^^^^^^^
+Bioconductor packages
+^^^^^^^^^^^^^^^^^^^^^
+
+Bioconductor packages are setup in a similar way to CRAN packages, but there
+are some very important distinctions. Bioconductor packages can be found at:
+https://bioconductor.org/. Bioconductor packages are R packages and so follow
+the same packaging scheme as CRAN packages. What is different is that
+Bioconductor itself is versioned and released. This scheme, using the
+Bioconductor package installer, allows further specification of the minimum
+version of R as well as further restrictions on the dependencies between
+packages than what is possible with the native R packaging system. Spack can
+not replicate these extra features and thus Bioconductor packages in Spack need
+to be managed as a group during updates in order to maintain package
+consistency with Bioconductor itself.
```

Comment:
We could simply create a bioconductor `BundlePackage` that uses the versioning scheme and package versions that Bioconductor uses. See the `xsdk` package for an example.

Reply:
I have thought of this, and think I suggested it in the past. I am on the fence about it though. First off, a bundle package would not address the fact that bioconductor packages would need to be updated as a group. It would not help with package creation, PR review, or updates. In fact, it would add one more level as new and updated packages would need to be added to the bundle. All of the bioconductor packages would need to be added with version specs to avoid the possibility of some packages picking up a newer version of a dependency if an older bioconductor bundle is installed. The package file would be quite large and ever growing. Finally, I am not sure that people really want to install all of the spack bioconductor packages, although that may be viewed as reasonable if the bundle package existed.

Reply:
I have thought of this, and think I suggested it in the past. I am on the fence about it though. First off, a bundle package would not address the fact that bioconductor packages would need to be updated as a group. It would not help with package creation, PR review, or updates. In fact, it would add one more level as new and updated packages would need to be added to the bundle. All of the bioconductor packages would need to be added with version specs to avoid the possibility of some packages picking up a newer version of a dependency if an older bioconductor bundle is installed. The package file would be quite large and ever growing. Finally, I am not sure that people really want to install all of the spack bioconductor packages, although that may be viewed as reasonable if the bundle package existed.